firmware/src/src/targets/nrf52/serial.cpp

```diff
@@ -191,7 +191,7 @@ void serial_Thread()
       if (rb_len) {
         int send_len = uart_fifo_fill(dev, buffer, rb_len);
         if (send_len < rb_len) {
-          LOG_ERR("USB CDC Ring Buffer Full, Dropped data");
+          // LOGE("USB CDC Ring Buffer Full, Dropped data");
```

Comment:
Is this chronic now, for cases where data goes out, but buffer files?  I wonder about the earlier trkset.stopAllData() thing that isn't on all edges of dtr...

Reply:
Crud.. this wasn't supposed to be in here. Must have copied it over from the other while I was doing serial things.